assets/js/src/public.jsx

```diff
@@ -85,6 +85,10 @@ jQuery(($) => {
     const form = formDiv.find('form');
     const position = form.data('position');
     formDiv.addClass(`mailpoet_form_position_${position}`);
+    const animation = form.data('animation');
+    if (animation !== '') {
+      formDiv.addClass(`mailpoet_form_animation_${animation}`);
+    }
```

Comment:
Do we have to do this using data attributes?
Why we just don't set the class in Twig template file here: https://github.com/mailpoet/mailpoet/blob/master/views/form/front_end_form.html#L13 ?
I find it as simpler and nicer solution.

The same applies to `position` two lines above.
Or is there some reason why this has to be done in JS?